package.json

```diff
@@ -31,16 +31,25 @@
     "pretest": "npm run lint",
     "test": "npm run testonly -- --coverage --forceExit --runInBand --verbose --bail",
     "coveralls": "nyc --instrument=false --source-map=false report --temp-directory=./coverage --reporter=text-lcov | coveralls",
-    "lint": "standard 'bin/*' 'client/**/*.js' 'examples/**/*.js' 'lib/**/*.js' 'pages/**/*.js' 'server/**/*.js' 'test/**/*.js'",
     "prepublish": "npm run release",
-    "precommit": "npm run lint"
+    "precommit": "lint-staged"
   },
   "standard": {
     "parser": "babel-eslint",
     "ignore": [
       "**/node_modules/**"
     ]
   },
+  "lint-staged": {
+    "*.js": [
+      "standard --fix",
```

Comment:
Just out of curiosity: why don't you want to fix automatically? In my experience this is a real time-saver.